docs/src/main/asciidoc/security-oidc-code-flow-authentication.adoc

```diff
@@ -1657,15 +1689,17 @@ public class CodeFlowAuthorizationTest {
 }
 ----
 
-`KeycloakTestResourceLifecycleManager` registers `alice` and `admin` users. The user `alice` has the `user` role only by default - it can be customized with a `keycloak.token.user-roles` system property. The user `admin` has the `user` and `admin` roles by default - it can be customized with a `keycloak.token.admin-roles` system property.
+`KeycloakTestResourceLifecycleManager` registers `alice` and `admin` users.
+The user `alice` has the `user` role only by default - it can be customized with a `keycloak.token.user-roles` system property.
+The user `admin` has the `user` and `admin` roles by default - it can be customized with a `keycloak.token.admin-roles` system property.
 
-By default, `KeycloakTestResourceLifecycleManager` uses HTTPS to initialize a Keycloak instance which can be disabled with `keycloak.use.https=false`.
-Default realm name is `quarkus` and client id - `quarkus-web-app` - set `keycloak.realm` and `keycloak.web-app.client` system properties to customize the values if needed.
+By default, `KeycloakTestResourceLifecycleManager` uses HTTPS to initialize a Keycloak instance that can be disabled by specifying `keycloak.use.https=false`.
+The default realm name is `quarkus` and client ID is `quarkus-web-app` - set `keycloak.realm` and `keycloak.web-app.client` system properties to customize the values if needed.
```

Comment:
```suggestion
The default realm name is `quarkus` and client id is `quarkus-web-app` - set `keycloak.realm` and `keycloak.web-app.client` system properties to customize the values if needed.
```

Reply:
I will change this back for consistency with feedback elsewhere, however, I do think "ID" is correct here. Perhaps we can revisit this discussion again at a later point.